docs/hc-dpki/seed.md

```diff
@@ -147,21 +147,21 @@ This root seed should be pure entropy
 ### rootSeed.init()
 delegate to base class
 
-**Kind**: instance method of [<code>RootSeed</code>](#RootSeed)  
+**Kind**: instance method of [<code>RootSeed</code>](#RootSeed)
 <a name="RootSeed+getDeviceSeed"></a>
 
 ### rootSeed.getDeviceSeed(index) ⇒ [<code>DeviceSeed</code>](#DeviceSeed)
 generate a device seed given an index based on this seed
 
-**Kind**: instance method of [<code>RootSeed</code>](#RootSeed)  
+**Kind**: instance method of [<code>RootSeed</code>](#RootSeed)
 
 | Param | Type |
 | --- | --- |
-| index | <code>number</code> | 
+| index | <code>number</code> |
 
 <a name="RootSeed.newRandom"></a>
 
 ### RootSeed.newRandom()
 Get a new, completely random root seed
 
-**Kind**: static method of [<code>RootSeed</code>](#RootSeed)  
+**Kind**: static method of [<code>RootSeed</code>](#RootSeed)
```

Comment:
looks like my IDE removed all the trailing whitespaces

Reply:
did you edit the docs directly? I don't think I hooked up the doc generation again after the lerna removal...

Reply:
yes, I did it manually as I forgot there was supposed to be automatic doc generation ^^''